.buildkite/steps/prepare-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env sh
+
+current=$(svu current)
+
+
+options() {
+  if [[ "$current" =~ "-rc" ]]; then
+    # if current version is 0.7.5-rc.1
+    # the options should be
+    # - 0.7.5
+    # - 0.7.5-rc.2
```

Comment:
Do you think this make sense?
I think if the current version is a pre-release, the option is either promote it as a _stable_ or increment the pre-release version.

Reply:
Yeah I think that makes sense :+1: